nemo/collections/common/tokenizers/text_to_speech/tts_tokenizers.py

```diff
@@ -521,12 +515,12 @@ def __init__(
         sep='|',  # To be able to distinguish between symbols
         add_blank_at=None,
         pad_with_space=False,
-        text_preprocessing_func=lambda text: english_text_preprocessing(text, lower=False),
     ):
         """General-purpose IPA-based tokenizer.
         Args:
             g2p: Grapheme to phoneme module, should be IPAG2P or some subclass thereof.
             punct: Whether to reserve grapheme for basic punctuation or not.
+            locale: Locale used to determine default text processing logic and punctuation.
```

Comment:
We should have a list of supported locales either here or in the documentation.

Reply:
I added the supported locales to the error message if someone provides a wrong one.

Should I provide a link in the docstring to the line or class path (nemo.collections.common.tokenizers.text_to_speech.ipa_lexicon) where the locale list is defined, even if that risks going out of date?

The best approach would be for the classes to have a "list_supported_locales()" method which a string that describes exactly what each locale string refers to, but that is probably overkill right now.

Reply:
Done